### PR TITLE
add discord share url

### PR DESCRIPTION
**Pull request by perwyl**

### CODE REVIEW

**Review by elfgoh — APPROVED** [review]
LGTM but I am unsure if the discord link in this PR never expires. 

But I do know Patricia configured https://discord.gg/XTw4CrRTpX to not expire. Please assess if this link is more appropriate before merging @perwyl

**Comment by serverwentdown** [comment]
See Discord chat for new URL

**Pull request merged**
